.github/workflows/node-ci.yml

```diff
@@ -0,0 +1,151 @@
+name: node-ci
+
+on:
+  workflow_dispatch:
+  push:
+    tags-ignore:
+      - '**'
+    paths-ignore:
+      - "platform/android/**"
+      - "platform/ios/**"
```

Comment:
You should probably also ignore Qt here.

Reply:
Good point 👍